docker-compose.yml

```diff
@@ -224,13 +224,16 @@ services:
       DISABLE_QUEUE_PUBLISHER: 1
       SENTRY_CURRENT_ENV: publishing-api-worker
       SENTRY_DSN: http://user:password@error-handler.dev.gov.uk/123
+    healthcheck:
```

Comment:
Why do we want to disable the health check having added it to the image?

Reply:
Ahh, forgot to mention in the commit message.  I'll update that now.  For the "worker" containers where we've overwritten the command to run ` sidekiq ` we're now disabling the healthcheck as it made no sense.  Docker was marking the container as "unhealthy", as HTTP was unreachable.

Reply:
Ah yep, makes sense. Wonder if disabling the worker health checks could be moved into a separate commit.

Reply:
Done.